packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -57,6 +73,53 @@ A ChatBot can share a link to a [quick start](/extensions/quick-starts) that wil
 ![Bot message that links a quick start tile.](./img/chatbot-quickstarts-tile.svg)
 </div>
 
+Selecting the quick start title, or the "Start" link can be configured to launch the quick start in a new window, as an interactive bot conversation within the ChatBot window, or as a separate tab within the ChatBot drawer (as shown in this [quick start/ChatBot concept demo](https://quickstart-chatbot-demo.surge.sh/?quickstart=install-app-and-associate-pipeline)).
+
+### Message actions 
+
+To allow users to interact with bots messages, utilize message actions. Refer to [the message actions React example](/patternfly-ai/chatbot/messages#message-actions) for an interactive visual.
```

Comment:
Is this a typo? Otherwise it's fine!

```suggestion
To allow users to interact with bot messages, utilize message actions. Refer to [the message actions React example](/patternfly-ai/chatbot/messages#message-actions) for an interactive visual.
```